test/unit/alphabet/cigar/cigar_test.cpp

```diff
@@ -56,3 +56,16 @@ TEST(cigar, assign_string)
     EXPECT_EQ(uint32_t{223}, seqan3::to_rank(get<0>(c1)));
     EXPECT_EQ('M',           get<1>(c1).to_char());
 }
+
+TEST(cigar, constexpr_char_literal)
+{
+    using seqan3::operator""_cigar_operation;
+    seqan3::cigar::operation op = 'M'_cigar_operation;
```

Comment:
```suggestion
    constexpr seqan3::cigar::operation op = 'M'_cigar_operation;
```

Reply:
I would change from 'M' as it is rank == 0; to 'D' which has rank 1.

```suggestion
    constexpr seqan3::cigar::operation op = 'D'_cigar_operation;
```